docs/usage/configuration.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Configuration
+description: How to configure Pester's behavior, options and features
```

Comment:
Unsure but.. `Pesters behavior`?

Reply:
In english I believe it's "Owner's item", but I'll rephrase it :)

Reply:
You could very well be correct, after re-reading the grammar rules I am now very confused :-D